tools/automation/tsconfig.json

```diff
@@ -1,9 +1,9 @@
 {
     "extends": "../../configs/base.tsconfig",
     "compilerOptions": {
-        "target": "es6",
```

Comment:
I don't think we should override the "target" value here unless there's a compelling reason.

Reply:
I removed the field